src/main/java/com/spartronics4915/frc2020/subsystems/LED.java

```diff
@@ -0,0 +1,100 @@
+package com.spartronics4915.frc2020.subsystems;
+
+import com.spartronics4915.lib.util.Logger;
+import com.spartronics4915.lib.subsystems.SpartronicsSubsystem;
+
+import edu.wpi.first.wpilibj.SerialPort;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ * The subsystem that controls the LED.
+ * Communicating to the Arduino by giving
+ * a byte using Serial to the wanted BlingState to be
+ * read by the Arduino to show the given Bling style wanted.
+ *
+ * Based off the Arduino code under Spartronics4915/Bling/Roborio_functionality
```

Comment:
Is this comment still correct? Or should it point to the Arduino folder in this same repository?

Reply:
that comment hurt my head... not sure what I? was thinking...